src/acme/pd/Ticket.java

```diff
@@ -118,6 +121,7 @@ public Company getCompany() {
 
     public void setCompany(Company company) {
         this.company = company;
```

Comment:
Same as above but wanted to leave a comment to remember to look at this.  

This will change when I get ticket creation finished but we shouldn't be hiding an auto add to the database if that is what is going on.

Reply:
This is not an auto add to the database, this is setting a field.

Reply:
Comment was 1 line too high